### ISSUE TITLE
Poor font rendering in Windows FullHD 150% font size

### ISSUE THREAD

**Issue by ncabanes** [geany/geany#1655]
I'm afraid this might be more related to GTK than Geany itself, but still...

Most of the computers I use have a 1366x768 screen resolution, and Geany is perfectly usable in them. But one of then has a FullHD screen in which text is too small, so I have set the Windows font size to 150%. Most applications render properly, and also do other editors such as Notepad++, or IDEs such as Visual Studio, but Geany results far less readable than in the other computers, which have 1366x768 and 100% font size.

I'm using Geany 1.31 and Windows 10 (in all the computers).

**Comment by elextr** [comment]
Yes thats GTK-2, GTK-3 I believe supports high DPI better.  If you can build Geany you could try it with GTK-3, but its kinda complex to do on Windows.

**Comment by fkbreitl** [comment]
I agree, the font rendering is not great and could be improved.

**Comment by fkbreitl** [comment]
I agree, the font rendering is not great and could be improved.
However, I found out that the situation improves significantly by changing (with right click on geany.exe):
`Geany Properties => Compatibility`
`'Overrite high DPI scaling behaviour' of Application`

I got the idea from [Fix Fuzzy Windows 10 Text on High Resolution Displays](https://www.groovypost.com/howto/fix-fuzzy-windows-10-fonts-high-resolution-displays/).

**Comment by RipNephils** [comment]
> I agree, the font rendering is not great and could be improved.
> However, I found out that the situation improves significantly by changing (with right click on geany.exe):
> `Geany Properties => Compatibility`
> `'Overrite high DPI scaling behaviour' of Application`
> 
> I got the idea from [Fix Fuzzy Windows 10 Text on High Resolution Displays](https://www.groovypost.com/howto/fix-fuzzy-windows-10-fonts-high-resolution-displays/).



I just made a GITHUB accoun to thank you for this. THANK YOU.

**Comment by fkbreitl** [comment]
@RipNephils Wow, thank you very much!!!
